### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by akshayn1107**
Lets get this ready for prod. Test went smoothly in stage.

@vikarmic @eogren

### CODE REVIEW

**Comment by vikarmic** [comment]
Looks pretty good to me.  Once the embargo is lifted tomorrow I think we're good to go, unless someone else has objections.

**Comment by akshayn1107** [comment]
I believe the embargo does not include gobblin since merging this has no implications on production code.

File: gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java [hunk 1]
```diff
@@ -383,7 +388,7 @@
   public static final String REST_SERVER_HOST_KEY = "rest.server.host";
   public static final String DEFAULT_REST_SERVER_HOST = "localhost";
   public static final String REST_SERVER_PORT_KEY = "rest.server.port";
-  public static final String DEFAULT_REST_SERVER_PORT = "8080";
+  public static final String DEFAULT_REST_SERVER_PORT = "8000";
```
Comment (eogren) [inline gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java]: is this change intentional?

File: gobblin-rest-service/gobblin-rest-server/src/test/java/gobblin/rest/JobExecutionInfoServerTest.java [hunk 2]
```diff
@@ -72,7 +72,7 @@ public void setUp()
     Injector injector = Guice.createInjector(new MetaStoreModule(properties));
     this.jobHistoryStore = injector.getInstance(JobHistoryStore.class);
 
-    this.client = new JobExecutionInfoClient("http://localhost:8080/");
+    this.client = new JobExecutionInfoClient("http://localhost:8000/");
```
Comment (eogren) [inline gobblin-rest-service/gobblin-rest-server/src/test/java/gobblin/rest/JobExecutionInfoServerTest.java]: and this one - intentional?

**Comment by eogren** [comment]
good to otherwise - as you said, don't need to worry about embargo since it won't affect prod anyways

File: gobblin-rest-service/gobblin-rest-server/src/test/java/gobblin/rest/JobExecutionInfoServerTest.java (hunk 2 above)
Comment (akshayn1107) [inline gobblin-rest-service/gobblin-rest-server/src/test/java/gobblin/rest/JobExecutionInfoServerTest.java]: Yeah. It interferes with our Jenkins port, although we might not be
compiling on Jenkins anymore so I might be able to revert this.

On Tuesday, May 12, 2015, Eric Ogren <notifications@github.com> wrote:

> In
> gobblin-rest-service/gobblin-rest-server/src/test/java/gobblin/rest/JobExecutionInfoServerTest.java
> <https://github.com/NerdWallet/gobblin/pull/18#discussion_r30152644>:
>
> > @@ -72,7 +72,7 @@ public void setUp()
> >      Injector injector = Guice.createInjector(new MetaStoreModule(properties));
> >      this.jobHistoryStore = injector.getInstance(JobHistoryStore.class);
> >
> > -    this.client = new JobExecutionInfoClient("http://localhost:8080/");
> > +    this.client = new JobExecutionInfoClient("http://localhost:8000/");
>
> and this one - intentional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NerdWallet/gobblin/pull/18/files#r30152644>.
>


-- 
Akshay Nanavati

Sent from mobile
